### ISSUE TITLE
Confusing terminology: "publication", "web publication"

### ISSUE THREAD

**Issue by jar398** [w3c/dpubwg-charter#24]
I find the Goals and Scope sections to be very hard to follow, in part because of the attempt to co-opt very common terms for the very specialized immediate purpose.

I see "publication" used in four different ways, none of which match the ordinary language meaning of the word. Similarly "web publication" is a common phrase and it seems to have little to do with what you're talking about.

Ordinary-language "publication" does not imply either "page-based" or "bounded", and ordinary-language "web publication" does not imply packaged or bundled with metadata.

I suggest you simply follow the practice that is common in most technical specifications. Coin a phrase that has not been previously used, so that it has no current connotations or associations. Then, imbue it with whatever meaning you like. Make an acronym if the phrase is too long.

For example, you could come up with an adjectival pharse A that is true of the artifacts you want to design but not true of ebooks in general, and then call these things A-ebooks, abbreviated AEB. (You can always change the name later on in the process; this is just a charter.)

A less attractive choice, used by FRBR with some success, is to choose a common word (like "expression") but to use it in contexts so far removed from its natural habitat that there is little chance of confusion. "Publication" and "web publication" definitely do not satisfy this condition, however.

Trying to redefine "publication" and "web publication" is fighting the tide, and provincial. It is bound to hurt your cause, not help it.

**Comment by lrosenthol** [comment]
@jar398 I've reviewed the Goals and Scope section and find that our use of the word publication is consistent and well defined.In fact, we define our usage of it up front and then continue to treat it in the same manner throughout.  Also, I find our definition is consistent with that provided by [standard dictionaries](http://www.dictionary.com/browse/publication?s=t).

In the same way, we very clear define what a web publication is and then how we see it being used in the works of this WG.  

Also, it should be pointed out that these terms are the ones used by this (publishing) community and to use a newly invented term would only cause confusion. 

If you have some specific editorial changes that we could make to improve the text to make it clearer for you, we welcome such specific changes.

**Comment by iherman** [comment]
@jar398 , also with respect to the comment of @lrosenthol, I would prefer to close this issue. Are you o.k. with this?

**Comment by jar398** [comment]
~~I don't care what you do with your~~ Do as you like with the issue tracker, but I care a little about being heard and understood, which I clearly wasn't. (That is probably mostly my fault, but not entirely, since communication is a two way street.) I only submitted the issue as what I thought was the desired way to respond to your widely disseminated request for comments on the draft charter. And I commented because I thought that an uninvolved outside perspective might be of some use to you (and to future readers of the TRs).

I can put some effort into sharpening my thoughts. Perhaps there is a better place to put them than the issue tracker?

**Comment by jar398** [comment]
hmm, that came out more strident than I intended.

**Comment by jar398** [comment]
Specific recommendations, as requested:

1. In order to avoid confusion with web publications as they are known in ordinary language today, replace "Web Publication" with "Web Publication Resource Set (WPRS)" throughout, where it is used "bare" i.e. not as a modifier and not modified. Or, some other expression of your choosing.

(I am not bothered by using "Web Publication" as a modifier or modified ("Packaged Web Publication"). But attempting to define bare "Web Publication" as a term of art is a disservice to the wider community that you claim to be trying to reach. Ordinary people are going to be using one of the terms you define, possibly this one, in the same way they sling around PDF or MP3 or eBook. If the common term ends up being 'PWP' (for Packaged Web Publication) or some other acronym that shows up in your final draft or reference implementation, and they never have to say "Web Publication" with capitals, then there will be no issue with collisions. But if there's any chance of a collision, now is the time to head it off.)

2. In order that the charter should have a prose style more appropriate to a W3C WG charter, remove the entire Goals section other than the the last sentence "It is the goal of the Publishing Working Group to ..." Then replace the end of that sentence (after "to") with specific common-sense technical targets, e.g. "to produce recommendations for a generalization of EPUB to be known as Web Publication Resource Sets, a package specification for such resource sets, and related documents as set out below, following the general principles laid out in [reference]."

Deleting that text makes any problem with the word "publication" go away.

But what about the deleted text? If you feel the Goals have to describe what the specs are to be used for, you could say something like the intended use being works that might be printed on paper, or some other common sense expression. If you think the Goals should say why the spec is needed, just say why other formats used for web publication (HTML, PDF, other EPUB versions, etc.) don't work for the applications you have in mind (I think you do spell that out somewhere?). Then a goal is to fix those deficiencies - that should be easy to write.  - However, for the most part, the Goals section of a WG charter is pithy, general, and technical, so probably not much detail or "selling" is needed at this point of the document.

**Comment by lrosenthol** [comment]
Thanks @jar398. Let me respond.

Let me start with some misunderstandings:
- A Web Publication is a general concept that we expect to apply to a set
of web resources that meet a set of criteria (to be determined by the WG as
part of its work) that are made available via the web/http.   These may or
may not be packaged and there may be multiple profiles of them.

- A Packaged Web Publication is also a general concept for what is
necessary to take a Web Publication and "package" it for use in
distribution "off the web".   Again, there will be multiple profiles of
them (of which EPUB 4 will be one).


As for goals, I believe it is important that we have this section for the
non-W3C people who will read it.  But I leave that to @iherman.

**Comment by TzviyaSiegman** [comment]
/me emerges from leave to comment that it's not up to the authors of the charter to tell readers what they don't understand. If readers say that this is unclear, then we must make it clear to those not involved in authoring the document.  I'll be back in May, but I do hope that this will be completed before I return.

**Comment by mattgarrish** [comment]
The name does pose a point of confusion between format and concept. If you accept that "web publication" is taken by a general audience as any publication on the web, then if you talk about a web publication are you talking about this format or the generic concept?

If the expectation is that this specification of "web publication" will eventually supplant the common understanding, then all is good. If not, then we'll get caught in a world of qualifiers in the future. Given the late start, the latter seems likely.

I don't like "resource set" as a name, as it's more like the lower-level epub package. Something like it will no doubt be needed at some point. If there is some other qualifier that could be swapped in it would be worth considering, but it seems like a number of options have already been tried on and failed.

I always liked "open web publication", to be honest, but it does have acronym baggage.

**Comment by iherman** [comment]
@jar398, you made two different proposals in https://github.com/w3c/dpubwg-charter/issues/24#issuecomment-289571074; to facilitate the discussion, I would prefer to separate these two in two different comments.

>  In order that the charter should have a prose style more appropriate to a W3C WG charter, remove the entire Goals section other than the the last sentence "It is the goal of the Publishing Working Group to ..." Then replace the end of that sentence (after "to") with specific common-sense technical targets, e.g. "to produce recommendations for a generalization of EPUB to be known as Web Publication Resource Sets, a package specification for such resource sets, and related documents as set out below, following the general principles laid out in [reference]."

I can see your point that this section may be a bit too long and general. On the other hand, I think that your proposal goes to the other extreme. I will try to come up with a somewhat shorter and more concise version, to see whether that can lead to a consensus. More about this later.

**Comment by iherman** [comment]
@jar398, this is my comment on:

> In order to avoid confusion with web publications as they are known in ordinary language today, replace "Web Publication" with "Web Publication Resource Set (WPRS)" throughout, where it is used "bare" i.e. not as a modifier and not modified. Or, some other expression of your choosing.

I must admit I am very hesitant of touching this. I fail to understand why the term "Web Publication" is worse than many others; we have spent a long time trying to pin this term and in my experience a discussion on such terminology can take an inordinate amount of time. I do find the "WPRS" term a mouthful and too 'technical'; I do not see how this term would be picked up by anyone, certainly not by "ordinary people".

As far as I am concerned, the term "Web Publication" expresses what we really mean as a general principle, namely to make digital publications using Web technologies first class entities on the Web. The term is of course not cast in concrete, and the WG may very well decide to come up with a better one in the course of its work (the charter does not forbid that).

My proposal is to keep that terminology as is, with an open mind as for how the WG proceeds. I am also fine adding a remark somewhere in the charter that makes it clear that this term may change in the course of the WG.

**Comment by iherman** [comment]
@jar398, @lrosenthol, @mattgarrish, @TzviyaSiegman

I have gone through the goal section of the charter. Re-reading it I actually tend to agree with @jar398 that this section needs a rewrite, it does not really fit the style of a charter.

I came up with a shorter text; I have created a separate Pull Request (#47) with the new section. You can also see the result in a human readable form at [rawgit](https://rawgit.com/w3c/dpubwg-charter/goal-section-rewrite/index.html), but I will reproduce the section below.

>Publications, from magazine articles to product documentation, from electronic books to scholarly journal articles are increasingly present on the Web. While following the traditions of the publishing world in terms of typesetting, layout, ergonomy, or usage patterns, the Web brings a new level of pervasive accessibility, internationalization, addressability, and interconnectedness to these publications. However, the combination of traditional publishing and the Web is not yet perfect: the <a href="https://www.w3.org/TR/pwp-ucr/">“Web Publications Use Cases and Requirements”</a> document, published by the <a href="https://www.w3.org/dpub/IG/">Digital Publishing Interest Group</a>, lists a number of use cases and requirements that are not yet covered by the current environments based on the Open Web Platform.
>
>It is the goal of the Publishing Working Group to provide the necessary technologies on the Open Web Platform to make the combination of traditional publishing and the Web complete in terms of the readers’ needs, portability, distribution, archiving, offline access, or reliable cross referencing. In short, _“publications”—with all their specificities and traditions—should become first-class entities on the Web_.

WDYT?

**Comment by mattgarrish** [comment]
The name just has a certain lack of specificity to it. It's sort of like Web Page to HTML in my mind. The web publication is the product, of course, but is it also the technology?

All I think it potentially leads to is people referring to them as "W3C Web Publications", or similar, in the future because "web publications" on its own could mean a variety of things depending on the audience's preconceptions.

But naming is a problem we can spin our wheels on endlessly, and we have spent a lot of time on it already. I'm fine with what we have to move the charter forward, especially if there's flexibility in the terminology and specification naming moving forward if we do see fit to make a change later.

**Comment by rickj** [comment]
as discussed on the call today, we are going to go forward with the existing terminology for web publications with the understanding that the group (once chartered) has the ability to make changes later, should a need be determined.

**Comment by rickj** [comment]
as discussed on the call today, we agreed to review the proposed edits to the goals section from Ivan above, and provide responses by mid week (Wednesday, local time) to allow this to be closed before the end of the week.

**Comment by rickj** [comment]
Other than having to lookup the word "ergonomy" to make sure it was what I thought, I'm supportive of the revised goals section in #47

**Comment by jar398** [comment]
Regarding "publishing" and the Goals section: #47 is a big improvement and I've commented there.

Regarding "Web Publication" I think some people are hearing me, so I will sign off.

**Comment by iherman** [comment]
In view of the last comment and #47, closing this issue.

**Comment by mac2net** [comment]
Hi Everyone
I have been away for a while, but in reviewing these logs I see there is STILL disagreement after a very very long time about a name for this format.
So I want to remind you I proposed something very different, but I believe it encompasses the group's objectives both physically and spiritually. 
**Browser Document Format (BDF)**

> The Browser Document Format, or BDF, is an Open Web Platform (OWP) workflow and replacement for EPUB & PDF.

https://medium.com/@Mac2net/bdf-browser-document-format-78ca50e7c534

Cheers
Mike

**Comment by rickj** [comment]
The issue I would have with "browser document format" is would cause confusion for offline, downloadable profiles of the format that may not be viewed in a browser, but in a native application instead.  Yes, I understand that technically they would probably be viewed using an embedded browser that the native app is using, but end users do not understand that nuance.

**Comment by mattgarrish** [comment]
Browser document format sounds like another name for HTML. If we make a change, it should add more specificity, not less.

**Comment by mac2net** [comment]
LOL browser is more appropriate than web. Web means internet. Browser doesn't. HTML is used for both. An embedded bowser is a browser. No problem. This has been going around in circles for almost a year. Let's commemorate! https://www.youtube.com/watch?v=ONST-MnTmYk

Honestly PWP, WP – they are awful names! And fully spelled out? Portable Web Publication... Then folks at XYZ and ZYX companies conspired to kill the first P because offline side-loading didn't fit their service worker cache "tie the customer in knots" agenda. Even EPUB+WEB was better...

Isn't it ironic that there is so much worry about accessibility while forcing users to go online to access content is the ultimate in _**inaccessibility**_!

The 4 pillars of BDF covers everything from screen to paper and the mechanisms to deliver this:
1. Open Web Platform HTML with additional optional features
2. Caching for pervasive access of browser content
3. Offline Url for a file and compressed containers (evolved from File Url)
4. Print CSS for a paginated and/or paper copy

**Comment by iherman** [comment]
Dear all, we have decided to close his issue because discussing on the terms/name can become an endless discussion which does not fundamentally change the charter. On the other hand, we are under a time pressure. For good or for worth, the term Web Publication has been accepted by the majority of those who were part of the discussions so far, and we have also acknowledged that the Working Group, if it gets accepted by the AC, will have the possibility to choose another name if it so wishes. I would propose to reserve our thoughts to the Working Group time.

**Comment by mac2net** [comment]
I keep seeing the name issue pop up, so, while we can close GitHub issues, the name debate is not going away. Also, since last fall, there have been 2 names - "PWP" and "WP", which is indicative of profound issues among the group members. "BDF" addresses this with it's multi-flavor approach in a coherent way.
